pkg/postgres/pgtest/conn/conn.go

```diff
@@ -25,7 +25,8 @@ func GetConnectionStringWithDatabaseName(database string) string {
 	}
 	pass := env.GetString("POSTGRES_PASSWORD", "")
 	host := env.GetString("POSTGRES_HOST", "localhost")
-	src := fmt.Sprintf("host=%s port=5432 user=%s database=%s sslmode=disable statement_timeout=600000", host, user, database)
+	port := env.GetString("POSTGRES_PORT", "5432")
```

Comment:
This could go as a separated PR

Reply:
I have moved it to another PR: https://github.com/stackrox/stackrox/pull/2612